### PR TITLE
classification metrics

### PR DESCRIPTION
**Pull request by ananyahjha93**
## What does this PR do?

adds classification metrics.

Fixes # (issue)

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/metrics/classification/f_beta.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+from collections import namedtuple
+
+import torch
+from torch import nn
+from pytorch_lightning.metrics.metric import Metric
+from pytorch_lightning.metrics.classification.precision_recall import _input_format
+from pytorch_lightning.metrics.utils import METRIC_EPS
+
+
+class Fbeta(Metric):
```
Comment (Borda) [inline pytorch_lightning/metrics/classification/f_beta.py:15]:
```suggestion
class FBeta(Metric):
```

File: pytorch_lightning/metrics/utils.py [hunk 2]
```diff
@@ -17,3 +20,31 @@ def dim_zero_mean(x):
 
 def _flatten(x):
     return [item for sublist in x for item in sublist]
+
+
+def to_onehot(
```
Comment (Borda) [inline pytorch_lightning/metrics/utils.py:25]: this is already in functional/classification

File: tests/metrics/classification/test_accuracy.py [hunk 3]
```diff
@@ -3,134 +3,92 @@
...
+from functools import partial
 
 from pytorch_lightning.metrics.classification.accuracy import Accuracy
 from sklearn.metrics import accuracy_score
 
-from tests.metrics.utils import compute_batch, NUM_BATCHES, BATCH_SIZE
+from tests.metrics.utils import compute_batch, setup_ddp
+from tests.metrics.utils import THRESHOLD
+
+from tests.metrics.classification.utils import (
```
Comment (Borda) [inline tests/metrics/classification/test_accuracy.py:14]: nice 👍 
maybe include in the function name that these functions are meant for testing
like `_testing_inputs_binary_prob`

File: tests/metrics/classification/test_f_beta.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    (
+            _testing_multidim_multiclass_inputs.preds,
+            _testing_multidim_multiclass_inputs.target,
+            _sk_fbeta_multiclass,
+            NUM_CLASSES,
+            False
+    )
+])
+@pytest.mark.parametrize("metric_class, beta", [(FBeta, 0.5), (FBeta, 1.)])
+def test_fbeta(
```
Comment (Borda) [inline tests/metrics/classification/test_f_beta.py:101]: I cannot find where is some comparison or our and ask metric, can you please make it more obvious...

**Review by Borda — CHANGES_REQUESTED** [review]

File: tests/metrics/classification/test_f_beta.py (hunk 4 above)
Comment (ananyahjha93) [reply tests/metrics/classification/test_f_beta.py:117]: test_f_beta.py contains fbeta_score metric in return values for sk_metric function
